integrations/gusto/mappers/to-standard-employee.ts

```
@@ -0,0 +1,121 @@
+            primaryAddress: {
+                street: '',
+                city: '',
+                state: '',
+                country: '',
+                postalCode: '',
+                type: 'WORK'
+            }
+        },
+
+        // Personal details
+        addresses: [] as Address[], // Gusto doesn't provide address details
```

ESLint

[error] Avoid casting objects. Add necessary type checks instead.

(@nangohq/custom-integrations-linting/no-object-casting)